clang/lib/Driver/ToolChains/Cuda.cpp

```diff
@@ -1162,6 +1162,20 @@ void CudaToolChain::AddIAMCUIncludeArgs(const ArgList &Args,
   HostTC.AddIAMCUIncludeArgs(Args, CC1Args);
 }
 
+llvm::SmallVector<ToolChain::BitCodeLibraryInfo, 12>
+CudaToolChain::getDeviceLibs(
```

Comment:
Kind ping @intel/dpcpp-clang-driver-reviewers -- any concerns about adding this method?